scripts/bundleSize/bundleSizeConfig.js

```diff
@@ -7,5 +7,5 @@
  * We are allowing a variance of -5 on `MIN_SIZE` and +5 on `MAX_SIZE` to avoid the need for frequent changes, as bundle sizes can fluctuate
  */
 
-export const MIN_SIZE = 645 - 5;
-export const MAX_SIZE = 1123 + 5;
+export const MIN_SIZE = 646 - 5;
+export const MAX_SIZE = 1125 + 5;
```

Comment:
Optional chaining basically. Optional chaining produces more code once transpiled, so will naturally increase the bundle size unfortunately!

Reply:
Ahhhh that's why! Thanks for explaining, definitely worth the hit as it is better than ramda's `path` or `pathOr` (🗑️ 🙏 )

Reply:
Yea it's a tricky one. May need to evaluate on a case by case basis and not use it at the top level of deeply nested objects, but rather further down so that it's not duplicating and checking null on the entirety of the object first.

Reply:
I may be wrong but I would expect any significant drop in bundle size. We are deleting code that should have been bundled into page specific bundles and even common bundles shared between CPS pages. Obviously as a collective there will no longer be a StoryPage, MediaAssetPage or PhotoGallery page bundle anymore.

Reply:
Ah yes, you're right. The largest page = Article Page & the largest language bundle = serbian, which is what the MAX_SIZE reflects. Sorry for the confusion!